tool/src/org/antlr/v4/codegen/Target.java

```diff
@@ -242,6 +243,7 @@ public String getTargetStringLiteralFromANTLRStringLiteral(
 	{
 		StringBuilder sb = new StringBuilder();
 
+		literal = escapeIfNeeded(literal);
```

Comment:
Could you explain why do you need escaping here? It doesn't look correct since the method always returns target string but not identifier.

Reply:
@KvanTTT You are right, there is no escaping needed here. I have removed it and added a test case for checking the escaping of reserve word `NULL` for cpp targets.